File: app/src/main/java/com/gmail/fattazzo/formula1world/ergast/json/objects/FinishingStatus.java
package com.gmail.fattazzo.formula1world.ergast.json.objects;

/**
 * @author fattazzo
 *         <p/>
 *         date: 06/06/17
 */
public class FinishingStatus {
    private int statusId;
    private int count;
    private String status;

    public FinishingStatus(int statusId, int count, String status) {
        this.statusId = statusId;
        this.count = count;
        this.status = status;
    }

    @Override
    public String toString() {
        return "FinishingStatus{" +
                "statusId=" + statusId +
                ", count=" + count +
                ", status='" + status + '\'' +
                '}';
    }
}

File: app/src/main/kotlin/com/gmail/fattazzo/formula1world/ergast/json/objects/FinishingStatus.kt
package com.gmail.fattazzo.formula1world.ergast.json.objects

/**
 * @author fattazzo
 *
 *
 * date: 06/06/17
 */
class FinishingStatus(private val statusId: Int, private val count: Int, private val status: String) {

    override fun toString(): String {
        return "FinishingStatus{" +
                "statusId=" + statusId +
                ", count=" + count +
                ", status='" + status + '\'' +
                '}'
    }
}
